int16   left_count
int16   right_count
int16   stasis_count
int16   dt_ms
time    reading_time
